Modules/getpath.c

```diff
@@ -1016,15 +1016,21 @@ calculate_zip_path(PyCalculatePath *calculate, const wchar_t *prefix)
             return PATHLEN_ERR();
         }
     }
-    status = joinpath(calculate->zip_path, L"lib/python00.zip", zip_path_len);
+    status = joinpath(calculate->zip_path, L"lib/python000.zip", zip_path_len);
     if (_PyStatus_EXCEPTION(status)) {
         return status;
     }
 
-    /* Replace "00" with version */
+    /* Replace "000" with version */
     size_t bufsz = wcslen(calculate->zip_path);
-    calculate->zip_path[bufsz - 6] = VERSION[0];
-    calculate->zip_path[bufsz - 5] = VERSION[2];
+    calculate->zip_path[bufsz - 7] = VERSION[0];
+    calculate->zip_path[bufsz - 6] = VERSION[2];
+    if (sizeof(VERSION) == 5) {
+      calculate->zip_path[bufsz - 5] = VERSION[3];
+    }
+    else {
+      memmove(&calculate->zip_path[bufsz - 5], &calculate->zip_path[bufsz - 4], bufsz);
```

Comment:
Limit lines to 79 chars ([PEP 7](https://www.python.org/dev/peps/pep-0007/#code-lay-out)).